src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
I'm actually not fond of the prefix, but it seems necessary if we need to import both the production class and its non-production extension (otherwise only one can be imported due to name collision, and the other would have to be referenced using its package name + class name).

Is it permissible to change the production classes to non-`final` and make the `private` constructors `protected` instead? This would allow the non-production version to extend the production class; only the non-production version would have to be imported then.

In summary, the options:

1. No prefix; use package name + class name when both production and non-production versions are referenced in the same test
2. No prefix; allow original util classes to be extended; only import non-production version into tests
3. Prefix all non-production versions; both can be imported normally into tests. It can also be a suffix instead of a prefix, e.g. `StringHelperExtension`?